presto-main/src/main/java/com/facebook/presto/cost/PlanNodeCostEstimate.java

```diff
@@ -46,7 +49,11 @@ public static PlanNodeCostEstimate networkCost(double networkCost)
     private final double memoryCost;
     private final double networkCost;
 
-    public PlanNodeCostEstimate(double cpuCost, double memoryCost, double networkCost)
+    @JsonCreator
```

Comment:
Why do these need to be json serializable? At what point do they leave the coordinator?

Reply:
It was done just so it can be included into the `PlanFragment`.

Reply:
Ok, so see my other comment about why are these being attached to the fragments vs the overall plan (since they are not a per-fragment entity)

Reply:
Even if we decided to do not attach this to `PlanFragment`, it still has to be serializable, since it hass to be passed to the `QueryMonitor` as part of the `QueryInfo` object, which is serizalizable.

Reply:
they don't, but fragments get serialized and deserialized somehow before explain.  Honestly, I didn't look into why that happens, I just saw the serialization errors when I tried running `EXPLAIN (TYPE DISTRIBUTED)` and fixed them.